devtools/td-layout-config/src/template/memory.jinja

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) 2021 - {{now() | date(format="%Y")}}  Intel Corporation
+//
+// SPDX-License-Identifier: BSD-2-Clause-Patent
+
+// Auto-generated by `td-layout-config`, do not edit manually.
+
+/*
+Mem Layout Example
+{%- for m in memory_regions | reverse%}
+{{true|format_layout_border}} <- {{m.region.end|format_hex}}
+|{{m.name | format_name}}|   ({{m.region.end - m.region.start|format_hex}}) {{m.region.end - m.region.start|filesizeformat}}
+{%- endfor %}
+{{true|format_layout_border}} <- {{memory_regions_base|format_hex}}
+Total memory: {{total_memory_length|format_hex}} ({{total_memory_length|filesizeformat}})
+*/
+
+// Runtime Layout Configuration
+{%- for m in memory_regions %}
+{%- if m.name != "FREE" %}
```

Comment:
why we need to hardcode the `FREE` region?

Reply:
Removed, add a special entry type to filter out the free region instead of using name